docs/get-started/lotus/installation.md

```diff
@@ -228,11 +228,11 @@ We recommend that MacOS users use [Homebrew](https://brew.sh) to install each of
 
 The `lotus` application runs as a daemon and a client to control and interact with that daemon. A daemon is a long-running program that is usually run in the background.
 
-When using _mainnet_, we recommend to start the daemon [syncing from a pruned snapshot](chain.md#syncing-from-a-pruned-snapshot-mainnet):
+When using _mainnet_, we recommend to start the daemon [syncing from a trusted state snapshot](chain.md#syncing-from-a-trusted-state-snapshot-mainnet):
 
 ```sh
 # For mainnet only:
-lotus daemon --import-snapshot https://very-temporary-spacerace-chain-snapshot.s3-us-west-2.amazonaws.com/Spacerace_pruned_stateroots_snapshot_latest.car
+lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car
```

Comment:
I thought we wanted to recommend users to sync from pruned state. This is much slower right?

Reply:
minimal == new name for pruned, sorry for the confusion